src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -282,6 +282,7 @@ public function getParent()
 
     public function reset()
     {
+        $this->idReaders = [];
```

Comment:
Oh: won't this prevent caching between requests thus reduce perf while it's not needed?

Reply:
Indeed, the readers are not depending on the request context, nor time sensitive. Closing here.